src/Nova/Contact.php

```diff
@@ -64,14 +73,14 @@ public function fields(Request $request)
                 FormType::PARTIES => 'Parties',
                 FormType::GROUPS => 'Groups',
                 FormType::EMPLOYMENT => 'Employment',
-            ])->required()->hideWhenUpdating(),
+            ])->rules('required')->hideWhenUpdating(),
```

Comment:
with this approch the validation is taken, with the `required()` not

Reply:
Sorry I don't understand this comment

Reply:
@joshtorres  with `->required()` doesnt show the validation properly... with rules('required') does